tests/integration/cairo1_hint_locals/deploy_contract_cairo0/deploy_contract_cairo0_test.cairo

```diff
@@ -20,9 +20,6 @@ fn test_deploy_contract_cairo0_w_ctor() {
 fn test_deploy_contract_cairo0_w_ctor_error() {
     let mut args = ArrayTrait::new();
     args.append(100);
-    match deploy_contract_cairo0('cairo0_w_ctor_error', args) {
-       Result::Ok(contract_address) => assert(false, 'should not have deployed'),
-       Result::Err(x) => assert(x == 88723383977659245052712844871367846102043870926066665245010961657188738359, x)
-    }
+    deploy_contract_cairo0('cairo0_w_ctor_error', args).unwrap();
```

Comment:
why u don't check the error here?

Reply:
Because cairo0 errors are basically un-parseable and errors are thrown in "basic" way
```
(500, {'code': <StarknetErrorCode.TRANSACTION_FAILED: 52>, 'message': 'Error at pc=0:108:\nAn ASSERT_EQ instruction failed: 0 != 1.\nCairo traceback (most recent call last):\nUnknown location (pc=0:121)'})
```

Reply:
So this one fails on unwrap